components/src/TransactionPendingFormConfirm/TransactionPendingFormConfirm.js

```diff
@@ -43,6 +44,7 @@ export default class TransactionPendingFormConfirm extends Component {
             className={ styles.confirmButton }
             fullWidth
             primary
+            disabled={ this.props.isSending }
             label='Confirm Transaction'
```

Comment:
it happens very fast, would just flicker (not cool)

we might wanna handle long responses in a separate pr, like if waiting time over 1-2 seconds, reflect sending state in UI